### ISSUE TITLE
luci-mod-system-mountpoints

### ISSUE THREAD

**Issue by 5p0ng3b0b** [openwrt/luci#5467]
TLDR:- No ubifs filesystem option in add/edit mount points advanced tab.
I have an extroot system and followed the [guide](https://openwrt.org/docs/guide-user/additional-software/extroot_configuration) to set it up. I noticed that despite mounting `/dev/ubi0_1` to `/rwm` using uci commands, it was not persistent despite there being an entry in `/etc/config/fstab`. I was in the mount points menu in luci and editing the custom mountpoint in the Advanced Settings tab. I noticed that in the filesystem option you have 'auto, ext2, ext3, ext4, squashfs'. I did not notice a 'ubifs' option. When using the mount command in a shell to mount a ubifs filesystem, I have to specify `-t ubifs` in the mount command otherwise it does not seem to consistantly work (perhaps auto recognising the filesystem from the busybox mount command iffy?) . I amended my `/etc/config/fstab` file and manually added `option fstype 'ubifs'` to fix the problem. I suppose I could have used the command `uci set fstab.rwm.fstype="ubifs"` but shouldn't ubifs be an option in the filesystem dropdown in luci?

**Comment by 5p0ng3b0b** [comment]
`cat /proc/filesystems` shows far more. The dropdown in luci is more like `cat /proc/filesystems | grep -v nodev | awk '{print $1}'`
